src/driftwood.js

```diff
@@ -0,0 +1,74 @@
+var _ = require('slapdash')
+var patterns = require('./patterns')
+var LEVELS = require('./levels')
+
+function noop () { }
+
+function createDriftwood (createConsoleLogger) {
```

Comment:
not a big fan of this createCreateLoggerCreatorCreator method but can't think of a better way